### DISCUSSION TITLE
Changing SOC UI cookie security / Accessing SO2 suite behind a NGINX Proxy

### DISCUSSION THREAD

**Discussion by jessepage1989** [Security-Onion-Solutions/securityonion#2800] [Unsupported Versions] [answered]
So I have SO16.04 set up to access from a NGINX proxy for a web address.  It works just fine.  When adding another entry for my SO2 instance it changes the domain name to the SO2 ip address.  I did a little digging in SO2 and was able to add my domainname to the NGINX instance in SO2. It fixes the problem of the ip address issue but now I have some cookie issue.  I believe at install you are suppose to designate how you are going to access the SOC.  How do you change these settings post installation?  "During install you must specify how you would like to access the SOC UI. This is for strict cookie security."  Changing entries in NGINX also are not persistent. There is some ansible script somewhere that overwrites any changes.

**Comment by TOoSmOotH** [comment, marked as answer]
url_base in the global.sls is what you should use. Change that to whatever your reverse proxy is.

**Comment by jessepage1989** [comment]
I changed the url_base in the file /opt/so/saltstack/local/pillar/global.sls and I get a ERR_TOO_MANY_REDIRECTS.  I also did a fresh install and put my FQDN in the portion that asks how to access the SOC. I still get a ERR_TOO_MANY_REDIRECTS. Cleared cookies like it recommends, no change. Firefox returns "This problem can sometimes be caused by disabling or refusing to accept cookies."

**Reply by jessepage1989** [reply]
It looks like it has something to do with the whoami/kratos settings.
